km_api/know_me/models.py

```diff
@@ -30,6 +30,26 @@ def get_gallery_item_upload_path(item, filename):
         id=item.profile.id)
 
 
+def get_km_user_image_upload_path(km_user, imagename):
+    """
+    Get the path to upload the kmuser image to.
+
+    Args:
+        km_user:
+            The km_user whose image is being uploaded.
+        imagename (str):
+            The original name of the image being uploaded.
+
+    Returns:
+        str:
+            The original image filename prefixed with
+            `users/<user_id>/{file}`.
+    """
+    return 'users/{id}/{file}'.format(
```

Comment:
Thinking about this some more, we probably want to make it as easy as possible when we move Friends & Faces over. That means this path should be something like `know-me/users/{id}/images/{file}`.